README.md

```diff
@@ -1,2 +1,51 @@
-# reviewapps-deploy-status
-A Github Action that tests the deploy status of a Heroku Review App
+# Review Apps deployment status
+A Github Action that tests the deploy status of a Heroku Review App.
+
+## Usage:
+* Include the action in the workflow
+    ```yaml
+    name: Review App Test
+
+    on:
+        pull_request:
+            branches:
+            - master
+
+    jobs:
+    review-app-test:
+
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v1
+        - name: Run review-app test
+        uses: #TODO: After publishing the action
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            span: 10 # optional in seconds
+            statuses: 200, 302, 403 # optional
+    ```
+
+> Note: Work flow should include `pull_request` event.
+
+* `GITHUB_TOKEN` is required to communicate with Github Deployment API.
```

Comment:
Do different tokens have different levels of permissions? If yes, we need to state which permissions are required for this action to work.

Reply:
Provided info regarding GITHUB_TOKEN.